sdk/src/azure/iot/az_iot_common.c

```diff
@@ -3,18 +3,135 @@
 
 #include <stdint.h>
 
-#include <azure/iot/az_iot_common.h>
-#include <azure/iot/internal/az_iot_common_internal.h>
-#include <azure/core/internal/az_precondition_internal.h>
 #include <azure/core/az_result.h>
 #include <azure/core/az_span.h>
+#include <azure/core/internal/az_precondition_internal.h>
 #include <azure/core/internal/az_span_internal.h>
+#include <azure/iot/az_iot_common.h>
+#include <azure/iot/internal/az_iot_common_internal.h>
 
 #include <azure/core/internal/az_log_internal.h>
 #include <azure/core/internal/az_retry_internal.h>
 
 #include <azure/core/_az_cfg.h>
 
+static const az_span hub_client_param_separator_span = AZ_SPAN_LITERAL_FROM_STR("&");
+static const az_span hub_client_param_equals_span = AZ_SPAN_LITERAL_FROM_STR("=");
+
+AZ_NODISCARD az_result az_iot_message_properties_init(
```

Comment:
I am going to assume all these were moved/renamed without any other source changes.

Reply:
Yes this is a find and replace all and/or move code as is. No additional code was written here.